### DISCUSSION TITLE
Tray menu not showing up after a while on Windows

### DISCUSSION THREAD

**Discussion by doorbash** [wailsapp/wails#2148] [General]
App tray menu does not show up after some time on Windows.  I think system sleep has something to do with it . I was not able to reproduce it happens randomly.

https://github.com/doorbash/my-process-manager/blob/1628b5bbeb9eb32b284caa8914156045df5675e1/app.go#L51

**Comment by leaanthony** [comment]
We don't support systray as a library. We are supporting tray menus natively and there's a PR open for comments.

**Reply by doorbash** [reply]
@leaanthony sorry my bad. wrong repo!
